balloc/ut/balloc.c

```diff
@@ -112,18 +112,26 @@ int test_balloc_ut_ops(struct m0_be_ut_backend *ut_be, struct m0_be_seg *seg,
 	int                     rc;
 	time_t                  now;
 
+	struct m0_ad_balloc_format_req bcfg;
+
 	time(&now);
 	srand(now);
 
+
```

Comment:
Remove this extra empty line.

Reply:
Taken care, left one extra empty line between declaration and initialization

Reply:
Done..